.swiftlint.yml

```diff
@@ -1,59 +1,67 @@
 disabled_rules:
-    - function_body_length
-    - type_body_length
-    - file_length
-    - generic_type_name
-    - force_cast
-    - force_try
-    - identifier_name
-    - type_name
-    - todo
-    - large_tuple
-    - unused_setter_value
-    - cyclomatic_complexity
-    - weak_delegate
-    - inclusive_language
-    - class_delegate_protocol
-    - nesting
-    - function_parameter_count
-    - notification_center_detachment
-    - discarded_notification_center_observer
-    - closure_parameter_position
-    - statement_position
-    - multiple_closures_with_trailing_closure
-    - xctfail_message
-    - compiler_protocol_init
-    - reduce_boolean
-    - legacy_cggeometry_functions
-    - legacy_constructor
-    - legacy_hashing
-    - no_fallthrough_only
+  - function_body_length
+  - type_body_length
+  - file_length
+  - generic_type_name
+  - force_cast
+  - force_try
+  - identifier_name
+  - type_name
+  - todo
+  - large_tuple
+  - unused_setter_value
+  - cyclomatic_complexity
+  - weak_delegate
+  - inclusive_language
+  - class_delegate_protocol
+  - nesting
+  - function_parameter_count
+  - notification_center_detachment
+  - discarded_notification_center_observer
+  - closure_parameter_position
+  - statement_position
+  - multiple_closures_with_trailing_closure
+  - xctfail_message
+  - compiler_protocol_init
+  - reduce_boolean
+  - legacy_cggeometry_functions
+  - legacy_constructor
+  - legacy_hashing
+  - no_fallthrough_only
+
+custom_rules:
+  todo_requires_jira_link:
+    name: "TODO: Requires Jira Link"
+    regex: '(TODO|FIX\s?ME|FIXME): (?!.*\\[WPB-\\d+\\])'
+    match_kinds: [comment]
+    message: "Do not add TODOs or FIX MEs without a JIRA reference. Include the JIRA ID like [WPB-680]."
```

Comment:
Maybe it's shorter and more encouraging (positive way to say it 😁):
`"Add JIRA reference to TODOs and FIX MEs like [WPB-680]."`

Reply:
Done!